### DISCUSSION TITLE
Index of Changepoint and Signal

### DISCUSSION THREAD

**Discussion by yasirroni** [deepcharles/ruptures#301] [Q&A]
What is the appropriate sign at boundary `<=` or `<`?

Given a signal with a length 10:

```
[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
```

Assume that we got breakpoints [2, 5, 10]

Thus, if I want to group it, what is the appropriate output? In [`jenkspy`](https://github.com/mthh/jenkspy/blob/20da64dc377d185e1604c2324bd7e75d7b6e1cdd/jenkspy/core.py#L104-L125), it will be `<=` 

```
[0, 1, 2]
[3, 4, 5]
[6, 7, 8, 9]
```

**Comment by yasirroni** [comment]
After digging some of the code, I found this: https://github.com/deepcharles/ruptures/blob/3d9d397a4deb1cb81cff852d07534974c5ad09f4/src/ruptures/detection/dynp.py#L45-L48

It means that the result will be:

```
[0, 1]
[2, 3, 4]
[5, 6, 7, 8, 9]
```

*I'm waiting for clarification for now. Thank you.

**Reply by deepcharles** [reply]
Yes, this is it.

**Reply by yasirroni** [reply]
Thanks.
